docs/system_overview/configuration.md

```diff
@@ -199,6 +193,10 @@ This file defines the behavior of the server Agent that answers individual reque
         // ... Additional Entries ...
     ],
 
+    // The fully qualified domain name or hostname which the server identifies as.
```

Comment:
can we rephrase to not end with a preposition?

not convinced this is the answer....
```
The fully qualified domain name or hostname which identifies the server 
```

Reply:
Yeah, I don't know that the suggestion indicates what the property means. The property defines what the server would return if `hostname` were a command provided by the iRODS API. Another way to look at it is - all invocations of `gethostname()` in the server code should be replaced by the value of this property.

Does that help in explaining what I'm attempting to capture?

Reply:
yes, i understand what it's supposed to be.  just that the grammar was weird/awkward.

Reply:
Hmm, maybe the suggestion IS what I'm looking for? The `host` property just needs to resolve to the server since one of its uses is to allow the server to connect to itself.

Reply:
From the server code:

> // The host property in server_config.json defines the true identity of the local server.
// We cannot use localhost or the loopback address because the computer may have multiple
// network interfaces and/or hostnames which map to different IPs.

Reply:
How about this?
```
The FQDN or hostname which represents the true identity of the server.
```

Reply:
On second thought, your suggestion is good. The important thing is to capture the rules around it. That is - no localhost, must resolve to the local machine for reasons.

Will update.

Reply:
Done.